app/presenters/teams/filtered_list.rb

```diff
@@ -19,8 +20,21 @@ def teams
       @current_user.teams
     end
 
-    def team_ids
-      list_param(:team_ids)
+    def team_id
+      @params[:team_id]
+    end
+
+    def filter_by_query
+      teams.includes(:folders, folders: [:accounts]).where('lower(teams.name) LIKE :query
```

Comment:
- formatting ?
- you're using lower and downcase method ?
- what about spaces ? maybe use strip ?